modules/hitl/src/views/full/message/index.jsx

```diff
@@ -68,29 +105,19 @@ export default class Message extends React.Component {
   renderMessage() {
     const date = moment(this.props.content.ts).format('DD MMM YYYY [at] LT')
 
-    const tooltip = <Tooltip id="tooltip">{date}</Tooltip>
-
     if (this.props.content.direction === 'in') {
       if (this.props.content.type === 'visit') {
-        return (
-          <OverlayTrigger placement="auto" overlay={tooltip}>
-            {this.renderMessageFromSystem()}
-          </OverlayTrigger>
-        )
+        return this.renderMessageFromSystem()
       }
 
-      return (
-        <OverlayTrigger placement="right" overlay={tooltip}>
-          {this.renderMessageFromUser()}
-        </OverlayTrigger>
-      )
+      return this.renderMessageFromUser()
+    } else {
```

Comment:
I'd use an else if instead and remove the last else:

```js
} else if (this.props.content.source == 'agent') {
    return this.renderMessageFromAgent()
}

return this.renderMessageFromBot()
```

-- or --

```js
} else if (this.props.content.source == 'agent') {
    return this.renderMessageFromAgent()
} else {
    return this.renderMessageFromBot()
}
```

Reply:
@rndlaine Updated!